packages/adapter-vercel/files/edge.js

```diff
@@ -15,6 +15,14 @@ const initialized = server.init({
 export default async (request, context) => {
 	await initialized;
 
+	const pathname = request.headers.get('REWRITE_HEADER');
```

Comment:
why not just make this an actual const, rather than using `replace`?

```suggestion
	const pathname = request.headers.get(REWRITE_HEADER);
```

Reply:
as discussed offline - because we cannot import `utils` here and that way everything is in sync leading back to that one const defintion within utils.